components/system/components/GlobalModal.js

```diff
@@ -1,4 +1,5 @@
 import * as React from "react";
+import FocusLock from "react-focus-lock";
```

Comment:
Nit: @akuokojnr can you put this import in better order on the imports. Add a new line, and put it underneath the `*` imports.